Chummer/Forms/EditGlobalSettings.cs

```diff
@@ -1522,8 +1522,16 @@ private void PopulateXsltList()
                                          : strFile,
                                      xmlSheet.SelectSingleNodeAndCacheExpression("name")?.Value ?? string.Empty));
                 }
-
-                string strOldSelected = cboXSLT.SelectedValue?.ToString() ?? string.Empty;
+                string strOldSelected;
+                try
+                {
+                    strOldSelected = cboXSLT.SelectedValue?.ToString() ?? string.Empty;
+                }
+                catch(IndexOutOfRangeException)
```

Comment:
Wait, what is this for?

Reply:
You can remove that if you want. Was in my branch because I had a selectedValue of 111 (out of range) and just wanted the program not to crash because of that. Didn't bother tough, WHY the value was that high.